doc/source/index.rst

```diff
@@ -1,3 +1,7 @@
+
+Ansys actions
```

Comment:
This is already included from the `README.rst`, right?

Reply:
Oh! I see it is no longer present. Thanks for adding this.

Reply:
https://github.com/ansys/actions/pull/588 with this PR, the directive starts after the title, so there is no title for the action in the stable docs
![image](https://github.com/user-attachments/assets/478d8b13-e493-4243-8b07-7048a9d5d8ab)